Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -472,12 +473,18 @@ export async function unlockKeychain(keychainPath: string, keychainPwd: string):
  * @param uuid
  */
 export async function deleteProvisioningProfile(uuid: string): Promise<void> {
-    let provProfilePath: string = getProvisioningProfilePath(uuid);
-    tl.warning('Deleting provisioning profile: ' + provProfilePath);
-    if (tl.exist(provProfilePath)) {
-        let deleteProfileCommand: ToolRunner = tl.tool(tl.which('rm', true));
-        deleteProfileCommand.arg(['-f', provProfilePath]);
-        await deleteProfileCommand.exec();
+    if (uuid) {
```

Comment:
trim ahead of checking emptiness.

Reply:
What if uuid is undefined? This should catch undefined, null and empty as well.

Reply:
Good point. Unfortunately if uuid is set to whitespace, it will be trimmed to nothing and we'll delete `*`.